lib/liquid/extensions.rb

```diff
@@ -7,6 +7,12 @@ def to_liquid
   end
 end
 
+class Symbol # :nodoc:
+  def to_liquid
+    self
```

Comment:
There are a few type checks for `String` in lib/liquid/utils.rb which won't match for a Symbol, however, the distinction is probably more likely to affect application code.

I think that doing `to_s` would be better to do here if we never want to treat Symbol as being different from String.  That seems like it would be better for keeping the language simpler and to avoid having to worry about the edge case of a Symbol being assigned to a variable.